IntelFsp2Pkg/Tools/fsp_tools_path_env.json

```diff
@@ -0,0 +1,4 @@
+{
+  "scope": "intel_fsp",
+  "flags": ["set_path", "set_pypath"]

```

Comment:
Do you need both path and pypath?

Reply:
Update in fifth Commit df8c67ce8a129fc03d9c174f3e246404fb05736f